tests/shell/tests/01-run.py

```diff
@@ -74,7 +74,7 @@
     ('echo escaped\\ space', '"echo""escaped space"'),
     ('echo escape within \'\\s\\i\\n\\g\\l\\e\\q\\u\\o\\t\\e\'', '"echo""escape""within""singlequote"'),
     ('echo escape within "\\d\\o\\u\\b\\l\\e\\q\\u\\o\\t\\e"', '"echo""escape""within""doublequote"'),
-    ("""echo "t\e st" "\\"" '\\'' a\ b""", '"echo""te st"""""\'""a b"'),
+    ("""echo "t\e st" "\\"" '\\'' a\ b""", '"echo""te st"""""\'""a b"'),  # noqa: W605
```

Comment:
Not sure I like suppressing linting on this very unreadable string. :-/

Reply:
Or rather: hard to parse string

Reply:
Isn't the string supposed to be hard toparse?

Reply:
I mean by the human eye, not the shell tokenizer ^^"

Reply:
So: `Invalid escape sequence 'x' (W605)` is the issue, can this be fixed by using raw strings and escape what is needed @aabadie ?

Reply:
You can't escape in raw strings, that's the point about them. Let me think about it.

Reply:
Github did make this its own review. See https://github.com/RIOT-OS/RIOT/pull/15373#pullrequestreview-524096821 why I resolved this conversation.